src/theme/Playground/styles.module.css

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.playgroundContainer {
+  margin-bottom: var(--ifm-leading);
+  border-radius: var(--ifm-global-radius);
+  box-shadow: var(--ifm-global-shadow-lw);
+  overflow: hidden;
+  border: 2px solid var(--ifm-color-emphasis-600);
+}
+
+.playgroundHeader {
+  letter-spacing: 0.08rem;
+  padding: 0.75rem;
+  text-transform: uppercase;
+  font-weight: bold;
+  background: var(--ifm-color-emphasis-600);
+  color: var(--ifm-color-content-inverse);
+  font-size: var(--ifm-code-font-size);
+  transition: max-height 1s ease-in-out;
```

Comment:
I think it'll be nicer if the animation does not take so long. Between 150ms and 300ms will make the UI feel snappier

Reply:
fixed